Tests/PinEdgesSpec.swift

```diff
@@ -565,6 +621,22 @@ class PinEdgesSpec: QuickSpec {
                 aView.pin.bottom(20).vertically()
                 expect(Pin.lastWarningText).to(contain(["vertically() bottom coordinate", "won't be applied", "already been set to 20"]))
             }
+            it("should warn") {
+                aView.pin.top(10).vertically(20)
```

Comment:
Once you have updated the method context, thanks to updating expectations, here it should contain `top`.